src/api/services/transformAllDocuments.js

```diff
@@ -9,7 +9,11 @@ export default async function transformAllDocument(
     job,
 ) {
     let handled = 0;
-    while (handled < count && (!job || (job && (await job.isActive())))) {
+    const isJobActive = async () =>
+        !job || (job.isActive && (await job.isActive()));
+    while (handled < count) {
+        if (!(await isJobActive())) break;
```

Comment:
Did we eslint this? Extra parentheses and no brackets looks weird to me?

Reply:
This is eslint(with prettier) that added the extra parentheses, I add brackets for clarity. I find it weird to...